src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -59,4 +56,10 @@
         public final static int LIFT_LIMIT_SWITCH_BOTTOM = 1;
         public final static int LIFT_LIMIT_SWITCH_TOP = 2;
     }
+
+    public static class Preferences {
+        //changes preferences into strings
+        public static final String GO_DOWN_SPEED = "speedWhenGoingDown";
```

Comment:
`LIFT_DOWN_SPEED`, and the string should probably be something like "Lift down speed"